### DISCUSSION TITLE
Version 1.14.0 does not support hp 22cw

### DISCUSSION THREAD

**Discussion by lufengQwQ** [xanderfrangos/twinkle-tray#381] [Monitor Compatibility Issues]
Hi
I found the version 1.14.0 does not support hp 22cwa, but version 1.13.11 does. 
Please fix and thank you for the software.

**Comment by xanderfrangos** [comment]
Hi @lufengQwQ,

Would you mind activating these 3 toggles under General -> Troubleshooting and then restart Twinkle Tray. Let me know if that helps at all. 

![Screenshot 2022-07-09 095118](https://user-images.githubusercontent.com/33106561/178108740-882c4039-273f-4ea4-8aa8-42b77451441a.png)

**Reply by lufengQwQ** [reply]
Only when I opened WMIC and WMI-Bridge. Very smoothly solved the problem. The DDC/CI function is also turned on. But can not use the DDC/CI function and the software can not find the monitor after using.
![Snipaste_2022-07-10_01-29-28](https://user-images.githubusercontent.com/75971722/178137658-b6eb142b-6bfc-4db8-a0bc-c0fd0eea4ffe.png)
![Snipaste_2022-07-10_01-31-15](https://user-images.githubusercontent.com/75971722/178137660-43d526bb-825e-400d-872e-bf51337c046b.png)
